lib/cn/kvset.c

```diff
@@ -2416,7 +2446,8 @@ kvset_iter_free_buffers(struct kvset_iterator *iter, struct kblk_reader *kr)
     vlb_free(kr->kr_buf[1].kmd_buf, kr->kr_buf[1].kmd_used_sz);
 
     if (iter->vreaders) {
-        for (i = 0; i < iter->ks->ks_vgroups; i++)
+        uint32_t nvgroups = kvset_get_vgroups(iter->ks);
+        for (i = 0; i < nvgroups; i++)
```

Comment:
Leave a blank line between decls and code...

Reply:
I think for vars scoped at function level this makes sense.  for vars in short code blocks like this it's more disruptive to code flow to force a blank line.

Reply:
A single blank line after decls give the reader a break and makes it easy to distinguish the change in context...

Reply:
you won't convince me

Reply:
We should have a rock, paper, scissors tournament to decide all aspects of our coding style.

Reply:
Yeah, I know.  But here's what's weird, Alex, AFAICT you often leave a blank line after a for statement before the decls/code, or maybe that was some clang-format code hoarking???

Reply:
Eh.  Not conciously.  But my code is art.  Sometimes it needs a format the matches the artistic intent.

Reply:
Blank lines to match the blank space in your head?? Just kidding! :)